libcloud/httplib_ssl.py

```diff
@@ -163,9 +163,10 @@ class LibcloudConnection(LibcloudBaseConnection):
     response = None
 
     def __init__(self, host, port, **kwargs):
-        self.host = '{0}://{1}'.format(
+        self.host = '{0}://{1}{2}'.format(
             'https' if port == 443 else 'http',
```

Comment:
@Kami yes, absolutely. That's a separate PR. I'm back to making one change at a time!